.github/workflows/unit_tests_with_woodwork_main_branch.yml

```diff
@@ -38,7 +38,7 @@ jobs:
       - name: Install Woodwork & Featuretools - test requirements
         run: |
           python -m pip install -e unpacked_sdist/[test]
-          python -m pip install -U git+https://github.com/alteryx/woodwork
+          python -m pip install --upgrade --force-reinstall https://github.com/alteryx/woodwork/archive/main.zip
```

Comment:
I don't think we need `--upgrade` if we are using `--force-reinstall`

Reply:
Removed